command/command.go

```diff
@@ -135,118 +157,70 @@ func PrintableCommandArgs(isQuoteFirst bool, fullCommandArgs []string) string {
 	return strings.TrimSpace(outStr), err
 }
 
-// RunCmdAndReturnTrimmedCombinedOutput ...
-func RunCmdAndReturnTrimmedCombinedOutput(cmd *exec.Cmd) (string, error) {
+// Deprecated: Use Command instead.
+func runCmdAndReturnTrimmedCombinedOutput(cmd *exec.Cmd) (string, error) {
 	outBytes, err := cmd.CombinedOutput()
 	outStr := string(outBytes)
 	return strings.TrimSpace(outStr), err
 }
 
-// RunCommandWithReaderAndWriters ...
-func RunCommandWithReaderAndWriters(inReader io.Reader, outWriter, errWriter io.Writer, name string, args ...string) error {
-	cmd := exec.Command(name, args...)
-	cmd.Stdin = inReader
-	cmd.Stdout = outWriter
-	cmd.Stderr = errWriter
-	return cmd.Run()
-}
-
 // RunCommandWithWriters ...
+// Deprecated: Use Command instead.
 func RunCommandWithWriters(outWriter, errWriter io.Writer, name string, args ...string) error {
 	cmd := exec.Command(name, args...)
 	cmd.Stdout = outWriter
 	cmd.Stderr = errWriter
 	return cmd.Run()
 }
 
-// RunCommandInDirWithEnvsAndReturnExitCode ...
-func RunCommandInDirWithEnvsAndReturnExitCode(envs []string, dir, name string, args ...string) (int, error) {
+// RunCommandWithEnvsAndReturnExitCode ...
+// Deprecated: Use Command instead.
+func RunCommandWithEnvsAndReturnExitCode(envs []string, name string, args ...string) (int, error) {
 	cmd := exec.Command(name, args...)
 	cmd.Stdin = os.Stdin
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
-	if dir != "" {
-		cmd.Dir = dir
-	}
 	if len(envs) > 0 {
 		cmd.Env = envs
 	}
 
-	return RunCmdAndReturnExitCode(cmd)
-}
-
-// RunCommandInDirAndReturnExitCode ...
-func RunCommandInDirAndReturnExitCode(dir, name string, args ...string) (int, error) {
-	return RunCommandInDirWithEnvsAndReturnExitCode([]string{}, dir, name, args...)
-}
-
-// RunCommandWithEnvsAndReturnExitCode ...
-func RunCommandWithEnvsAndReturnExitCode(envs []string, name string, args ...string) (int, error) {
-	return RunCommandInDirWithEnvsAndReturnExitCode(envs, "", name, args...)
+	return runCmdAndReturnExitCode(cmd)
 }
 
-// RunCommandInDir ...
-func RunCommandInDir(dir, name string, args ...string) error {
+// RunCommand ...
+// Deprecated: Use Command instead.
+func RunCommand(name string, args ...string) error {
 	cmd := exec.Command(name, args...)
 	cmd.Stdin = os.Stdin
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
-	if dir != "" {
-		cmd.Dir = dir
-	}
 	return cmd.Run()
 }
 
-// RunCommand ...
-func RunCommand(name string, args ...string) error {
-	return RunCommandInDir("", name, args...)
-}
-
-// RunCommandAndReturnStdout ..
+// RunCommandAndReturnStdout ...
+// Deprecated: Use Command instead.
 func RunCommandAndReturnStdout(name string, args ...string) (string, error) {
 	cmd := exec.Command(name, args...)
-	return RunCmdAndReturnTrimmedOutput(cmd)
+	return runCmdAndReturnTrimmedOutput(cmd)
 }
 
-// RunCommandInDirAndReturnCombinedStdoutAndStderr ...
-func RunCommandInDirAndReturnCombinedStdoutAndStderr(dir, name string, args ...string) (string, error) {
```

Comment:
Inlined because it was [unused](https://github.com/search?q=org%3Abitrise-io+org%3Abitrise-steplib+%22RunCommandInDirAndReturnCombinedStdoutAndStderr%22&type=Code) in external repositories.